price-feeder/price-feeder.example.toml

```diff
@@ -76,9 +76,9 @@ enable-hostname = true
 enable-hostname-label = true
 enable-service-label = true
 enabled = true
-global-labels = [["chain-id", "umee-local-testnet"]]
+global-labels = [["chain_id", "umee-local-testnet"]]
```

Comment:
The keys names should be underscores (_) instead of dashes (-) right?

Reply:
For this specifically we're using dashes (-), as the cosmos telemetry package we recently switched to uses dashes